docs/guides/other/troubleshooting.md

```diff
@@ -143,3 +143,82 @@ might print:
     'left_panel': <Slot component_name='layout' slot_name='left_panel'>,
 }
 ```
+
+## Agentic debugging
```

Comment:
This is awesome. I have this same workflow when working on another hobby-project, and it works great!